src/server/lib/example.js

```diff
@@ -6,3 +6,21 @@ exports.pluralize = function (count, singular, plural) {
   return `${count} ${plural}`;
 };
 
+
+exports.pluralize_v2 = function (count, singular, plural) {
+  if (count === 1) {
+    return `${count} ${singular}`;
+  }
+  plural = plural || `${singular}s`;
+  return `${count} ${plural}`;
+};
+
+exports.method = function (count) {
+  for(let i = 0; i < count; i++){
+    const j = 9
+    while(j>count){
+      console.log("counting")
+      j--
```

Comment:
Codacy found an issue: ['j' is constant.](https://app.codacy.com/gh/codacy-acme/nodeproject/pullRequest?prid=5794001)